plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/schema/SQLSchemaManager.java

```diff
@@ -111,6 +117,11 @@ public void updateSchema(DBRProgressMonitor monitor) throws DBException {
                             versionManager.getLatestSchemaVersion()
                         );
                     } else if (schemaVersionActual > currentSchemaVersion) {
+                        BackupDescriptor descriptor =
+                                BackupRegistry.getInstance().getCurrentDescriptor(this.targetDatabaseDialect);
+                        if (descriptor != null) {
+                            descriptor.getInstance().doBackup(dbCon, currentSchemaVersion, databaseConfig);
```

Comment:
Surround it with try/catch and re-throw  error like "Internal database backup has failed"